packages/core/content-manager/server/src/controllers/relations.ts

```diff
@@ -167,133 +244,78 @@ export default {
       addFiltersClause(queryParams, { id: { $notIn: knexSubQuery } });
     }
 
-    ctx.body = await strapi.entityService.findPage(targetedModel.uid, queryParams);
+    // We find a page of the targeted model (relation) to display in the
+    // relation select component in the CM edit view
+    const res = await strapi.entityService.findPage(
+      targetUid as Common.UID.ContentType,
+      queryParams
+    );
```

Comment:
I wonder if this would be the same as:
```suggestion
    const res = await strapi.db.query(targetUid).findPage(
      queryParams
    );
```

that way we would remove the dependency with the soon to be deprecated entityService